### PR TITLE
PXP-9113 Add 'previous_versions' prop to CMC node

### PR DESCRIPTION
**Pull request by paulineribeyre**
Jira Ticket: [PXP-9113](https://ctds-planx.atlassian.net/browse/PXP-9113)

### New Features
- Add 'previous_versions' prop to CMC node

### CODE REVIEW

**Review by hughestr — APPROVED** [review]

**Comment by paulineribeyre** [comment]
waiting for sponsor to approve in QA before merging

**Pull request merged**
